routes/originPyPi.js

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation and others. Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+const asyncMiddleware = require('../middleware/asyncMiddleware')
+const router = require('express').Router()
+const requestPromise = require('request-promise-native')
+
+router.get(
+  '/:namespace?/:name/revisions',
+  asyncMiddleware(async (request, response) => {
+    const { name } = request.params
+    const url = `https://pypi.python.org/pypi/${name}/json`
+    const answer = await requestPromise({ url, method: 'GET', json: true })
+    const result = answer && answer.releases ? Object.keys(answer.releases) : []
+    result.reverse()
+    return response.status(200).send(result)
+  })
+)
+
+router.get(
+  '/:namespace/:name?',
+  asyncMiddleware(async (request, response) => {
```

Comment:
Where is name used here? It should be the partial value being searched for.

Reply:
I think there is no namespace, just a name. For example a valid URL is: https://pypi.org/pypi/billiard/json

Reply:
OK. Interestingly confusing they do have something called `namespace packages` but that appears to be more a way of sharding packages and joining them into one at runtime. 
So @Teju-Manchenella if there are no namespaces, lets remove the namespace code from the pypi support.